draft-ietf-webtrans-http2.md

```diff
@@ -854,6 +854,77 @@ natively supports QUIC datagrams, such as WebTransport over HTTP/3
 {{WEBTRANSPORT-H3}}, intermediaries follow the requirements in
 {{WEBTRANSPORT-H3}} to use native QUIC datagrams.
 
+## CLOSE_WEBTRANSPORT_SESSION Capsule {#CLOSE_WEBTRANSPORT_SESSION_CAPSULE}
+
+*[CLOSE_WEBTRANSPORT_SESSION_CAPSULE]: #
+
+WebTransport over HTTP/2 uses the CLOSE_WEBTRANSPORT_SESSION capsule defined in
+{{Section 5 of WEBTRANSPORT-H3}} to terminate a WebTransport session with an
```

Comment:
Not sure if it's okay to have this reference the H3 spec, instead of the Overview doc, but IIRC that doesn't have capsule definitions, so maybe it's okay for now? (Same for the DRAIN capsule)

Reply:
Yeah, I don't think we want to duplicate the capsule definitions, but I'm not sure the overview doc is necessarily the place for them. Right now, we've kind of picked HTTP/3 to be the document that holds the capsules that overlap, but we could adjust that (or extract them out) as necessary. 

To fully address the needs of the W3C spec, though, we may want to introduce these as concepts more explicitly in the overview, if they're not there already. Added a note about that in https://github.com/ietf-wg-webtrans/draft-ietf-webtrans-overview/issues/12.